CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
 
 ## [Unreleased]
+### Added
```

Comment:
Isn't this a change? The previous cli took --response-code and now it takes --response-codes. It seems the implementation could wrap the --response-code cli option into a list and preserve both features enhancing expressiveness while maintaining backwards compatibility.

Reply:
If that is the case it would be a `### Breaking Changes` header (which would be a major version bump). In this case we have a couple of options to avoid the need to make a breaking change:
- avoid changing the option name, just update the description
- add a new option name, make the old option print a deprecation warning and do exactly what the new option does. We can then remove the old option in a major release. The benefit is guiding people to change it before it breaks you